### PR TITLE
NGRAPH-2762: Add DeepSpeed2

### PR DESCRIPTION
**Pull request by lnguyen-nvn**
## Description ##
PR to test inference of the DeepSpeech 2.
There are some modifies in this PR:
1. Clone https://github.com/baidu-research/warp-ctc.git inside the docker container (https://github.intel.com/AIPG/cje-algo/pull/259)
2. Modify the config.mk, and build Ngraph_Mxnet with warp-ctc.
2. Run the DeepSpeech2. 

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by lnguyen-nvn** [comment]
Please do not merge this PR since I need to merge https://github.intel.com/AIPG/cje-algo/pull/259 before the PR.

File: docker/scripts/lib_validation_testing.py [hunk 1]
```diff
@@ -362,6 +362,34 @@ def runBenchmarkScoreScript(script=None,          # Script to run
 
 # End: def runBenchmarkScoreScript()
 
+def runDSDeepMarkScript(sourceDir=None,
+                    script=None,          # Script to run
+                   logID='',
+                   ompNumThreads=None,
```
Comment (pruthviIntel) [inline docker/scripts/lib_validation_testing.py]: instead of setting `None` here, you can set them to default values.

Comment (lnguyen-nvn) [reply docker/scripts/lib_validation_testing.py]: I will make a enhance in the next PR to handle the valid input.
